### ISSUE TITLE
[Problem/Bug]: WebView2 page loaded for some but not for others using MSIX package.

### ISSUE THREAD

**Issue by richardjin** [MicrosoftEdge/WebView2Feedback#4383]
### What happened?

We have a windows standalone application that uses WebView2, developed in VS 17.6.5, for the release, we create a MSIX package. When users start to download it, some people can see the content in WebView2 without any issue, but some just see a bland screen, they can right-mouse click the WebView2 area and run "Inspect", but nothing display.

### Importance

Important. My app's user experience is significantly compromised.

### Runtime Channel

Stable release (WebView2 Runtime)

### SDK Version

1.0.2210.55

### Framework

Winforms

### Operating System

Windows 11

### OS Version

22621.3085

### Repro steps

The system works fine for some but not all users.

### Repros in Edge Browser

Yes, issue can be reproduced in the corresponding Edge version

### Regression

No, this never worked

### Last working version (if regression)

1.0.2210.55

**Comment by victorhuangwq** [comment]
Could you provide the runtime version? It should look something like 122.0.2365.38

**Comment by richardjin** [comment]
Victor,

Thanks for your quick response, the runtime version is 121.0.2277.128.

Thank you,

Richard Jin
Assistant Vice President/Senior Developer II
USLI Technology Team
888-523-5545 ext. 2178 | ***@***.*** | Alt. contact Belinda Roscoe ext. 2833
1190 Devon Park Drive, Wayne, PA 19087

[A picture containing text  Description automatically generated]
SPEED, ACCESSIBILITY and
EASE OF USETM

Learn more<https://customers.usli.com/sites/experience/usli-speed-accessibility-eou-us.html> or quote over the phone 888-773-8754,
online<https://customers.usli.com/sites/experience/usli-quote-online-us.html> or via email<https://customers.usli.com/sites/experience/usli-quote-email-us.html> submission. You can even reply to this email,
and I’ll make sure your submission gets to the right team!


From: Victor Huang ***@***.***>
Sent: Tuesday, February 20, 2024 5:55 PM
To: MicrosoftEdge/WebView2Feedback ***@***.***>
Cc: Richard Jin ***@***.***>; Author ***@***.***>
Subject: (EXTERNAL) Re: [MicrosoftEdge/WebView2Feedback] [Problem/Bug]: WebView2 page loaded for some but not for others using MSIX package. (Issue #4383)

CAUTION EXTERNAL EMAIL. This email originated outside of USLI.

Could you provide the runtime version? It should look something like 122.0.2365.38

**Comment by victorhuangwq** [comment]
@johna-ms I am triaging this as a hosting issue for now, but let me know if you think otherwise.

**Comment by victorhuangwq** [comment]
To use WebView2 in your application, you would need to install the WebView2 runtime on the device. I am assuming that's already covered?

**Comment by richardjin** [comment]
Yes, WebView2 runtime has been installed on all users’ machines.

It’s also weird that for regular users who don’t have Visual Studio the program is running fine; for users with VS, some are good, some bad.

Thank you,

Richard Jin
Assistant Vice President/Senior Developer II
USLI Technology Team
888-523-5545 ext. 2178 | ***@***.*** | Alt. contact Belinda Roscoe ext. 2833
1190 Devon Park Drive, Wayne, PA 19087

[A picture containing text  Description automatically generated]
SPEED, ACCESSIBILITY and
EASE OF USETM

Learn more<https://customers.usli.com/sites/experience/usli-speed-accessibility-eou-us.html> or quote over the phone 888-773-8754,
online<https://customers.usli.com/sites/experience/usli-quote-online-us.html> or via email<https://customers.usli.com/sites/experience/usli-quote-email-us.html> submission. You can even reply to this email,
and I’ll make sure your submission gets to the right team!


From: Victor Huang ***@***.***>
Sent: Friday, February 23, 2024 2:05 PM
To: MicrosoftEdge/WebView2Feedback ***@***.***>
Cc: Richard Jin ***@***.***>; Author ***@***.***>
Subject: (EXTERNAL) Re: [MicrosoftEdge/WebView2Feedback] [Problem/Bug]: WebView2 page loaded for some but not for others using MSIX package. (Issue #4383)

CAUTION EXTERNAL EMAIL. This email originated outside of USLI.

To use WebView2 in your application, you would need to install the WebView2 runtime on the device. I am assuming that's already covered?

**Comment by victorhuangwq** [comment]
Could there be multiple versions of the WebView2 runtime installed on the devices of the users who have VS?

**Comment by richardjin** [comment]
I did check on user’s machine using control panel, I didn’t see multiple versions

***@***.***

Thank you,

Richard Jin
Assistant Vice President/Senior Developer II
USLI Technology Team
888-523-5545 ext. 2178 | ***@***.*** | Alt. contact Belinda Roscoe ext. 2833
1190 Devon Park Drive, Wayne, PA 19087

[A picture containing text  Description automatically generated]
SPEED, ACCESSIBILITY and
EASE OF USETM

Learn more<https://customers.usli.com/sites/experience/usli-speed-accessibility-eou-us.html> or quote over the phone 888-773-8754,
online<https://customers.usli.com/sites/experience/usli-quote-online-us.html> or via email<https://customers.usli.com/sites/experience/usli-quote-email-us.html> submission. You can even reply to this email,
and I’ll make sure your submission gets to the right team!


From: Victor Huang ***@***.***>
Sent: Friday, February 23, 2024 2:14 PM
To: MicrosoftEdge/WebView2Feedback ***@***.***>
Cc: Richard Jin ***@***.***>; Author ***@***.***>
Subject: (EXTERNAL) Re: [MicrosoftEdge/WebView2Feedback] [Problem/Bug]: WebView2 page loaded for some but not for others using MSIX package. (Issue #4383)

CAUTION EXTERNAL EMAIL. This email originated outside of USLI.

Could there be multiple versions of the WebView2 runtime installed on the devices of the users who have VS?

**Comment by johna-ms** [comment]
@richardjin if all users have the runtime, and the rest of the app run correctly, and the context menu opens fine, I don't think this is a binary dependencies issue. To rule out VS as a factor, if possible, you can try installing VS on working environments and see if that breaks the app. Is there some pattern in OS version for working/non working cases? And you mention it matters that the app is installed via msix. If you distribute just the exe to non-working environments, does that work fine?
